apps/network-alerts/test/unit/attention/attention_test.js

```diff
@@ -103,11 +100,14 @@ suite('Network Alerts - Attention Screen', function() {
     });
 
     test('Notification should be displayed and alert is played', function() {
+      this.sinon.spy(NotificationHelper, 'send');
```

Comment:
Agree. @hkasemir  - Stub behaves like spy but is cheaper. Since the code you are stubbing is already a mock (i.e. doesn't perform any real action) you don't need to replace the original behavior, so just replace spy with stub.